thoth/python/source.py

```diff
@@ -125,13 +131,58 @@ def _warehouse_get_api_package_info(self, package_name: str) -> dict:
         response.raise_for_status()
         return response.json()
 
-    def _warehouse_get_package_hashes(self, package_name: str, package_version: str) -> typing.List[dict]:
+    def _construct_contents(self, path: str):
```

Comment:
It would be cool if you add `-> dict` as the other functions.

Reply:
@leonjia0112 Thanks a lot! I have added.